docs/production.md

```diff
@@ -11,3 +12,180 @@ configurations are available:
 | ----------------------- | ------------------------------- | -----------
 | OpenCensus Agent        | `OCAGENT`                       | Use OpenCensus.
 | Stackdriver\*           | `STACKDRIVER`                   | Use Stackdriver.
+
+
+## Rotating secrets
```

Comment:
I like this a lot, I look forward to the tooling. Maybe add a TODO for that?

Reply:
I was toying with the tooling yesterday. It's actually somewhat annoying because Terraform is managing the secret and the envvars.